except_star.md

```diff
@@ -1,31 +1,172 @@
 # Introducing try..except* syntax
 
-## Disclaimer
 
-* We use the `ExceptionGroup` name, even though there
-  are other alternatives, e.g. `AggregateException` and `MultiError`.
-  Naming of the "exception group" object is out of scope of this proposal.
+## Abstract
 
-* We use the term "naked" exception for regular Python exceptions
-  **not wrapped** in an `ExceptionGroup`. E.g. a regular `ValueError`
-  propagating through the stack is "naked".
+This PEP proposes language extensions that allow programs to raise and handle
+multiple unrelated exceptions simultaneously:
+
+* A new standard exception type, the `ExceptionGroup`, which represents a group
+of unrelated exceptions being propagated together.
+
+* A new keyword `except*` for handling `ExceptionGroup`s.
+
+## Motivation
+
+The interpreter is currently able to propagate at most one exception at a time.
+The chaining features introduced in PEP3134 [reference] link together exceptions
+that are related to each other as the cause or context, but there are situations
+where there are multiple unrelated exceptions that need to be propagated together
+as the stack unwinds. Several real world use cases are listed below.
+
+[TODO: flesh these out]
+* asyncio programs, trio, etc
+
+* Multiple errors from separate retries of an operation [https://bugs.python.org/issue29980]
```

Comment:
FWIW this exact issue is mirrored in the asyncio's `loop.create_connection()` API. So even without TaskGroups we have some examples where exception groups are needed in asyncio.